src/template-tags/tickets.php

```diff
@@ -123,6 +123,11 @@ function tribe_tickets_display_count_and_ticket_button() {
 		// get an array for ticket and rsvp counts
 		$types = Tribe__Tickets__Tickets::get_ticket_counts( $event_id );
 
+		// if no rsvp or tickets return
+		if ( ! $types ) {
+			return null;
```

Comment:
It can return null/void, but there's no point I see where it returns a string (it can echo a string, but that's not the same thing ... `@return string` is telling devs they can get back a string, which they can't)